svc/butler-srv/cmd/serve.go

```diff
@@ -0,0 +1,63 @@
+package cmd
+
+import (
+	"github.com/Azeite-da-Quinta/jigajoga/svc/butler-srv/srv"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var serveCmd = &cobra.Command{
+	Use:   "serve",
+	Short: "Serves the JWT  endpoint",
+	Long: `This is the main command of the app.
+It'll serve an HTTP and websocket server. Allows connecting players
+to generate JWT token to authorize connection`,
+	Run: func(cmd *cobra.Command, args []string) {
+		setLogLevel()
+
+		s := srv.Server{
+			Config: srv.Config{
+				Port:      viper.GetInt(port),
+				Version:   viper.GetString(version),
+				JWTSecret: viper.GetString(jwtsecret),
+				Mode:      viper.GetString(mode),
```

Comment:
I don't think you need the mode here. It exists in game-srv because it's a stateful app that behaves differently in sharded.